task-management/src/main/webapp/vue-app/tasks-management/components/ProjectTasks/TasksViewBoard.vue

```diff
@@ -69,9 +69,7 @@ import {updateTask} from '../../../taskDrawer/taskDrawerApi';
         return tasksByStatus;
       },
       updateTaskCompleted(e){
-        if( !this.filterTaskCompleted ) {
```

Comment:
This should not be removed IMO, please check when the filter hide completed tasks is not checked